lib/validators/plugin.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+import { UPDATE_COMPONENT, REMOVE_COMPONENT, CREATE_COMPONENTS } from '../component-data/mutationTypes';
+import { UPDATE_PAGE } from '../page-data/mutationTypes';
+import { OPEN_DRAWER } from '../drawers/mutationTypes';
+const relevantMutations = [
+    UPDATE_COMPONENT,
+    OPEN_DRAWER,
+    REMOVE_COMPONENT,
+    CREATE_COMPONENTS,
+    UPDATE_PAGE
+  ],
+  relevantDrawers = ['health', 'publish'],
+  validate = _.debounce(store => store.dispatch('validate'), 250);
```

Comment:
could you add a comment here on why there's debouncing? What you said in the PR comment is good.